cli/assets/generated/TestTemplate.t.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+import "../src/{contract_name}.sol"; 
+
+contract {contract_name}Test is Test {
+    {contract_name} public {instance_name};
+
+    function setUp() public {
+        {instance_name} = new {contract_name}();
```

Comment:
Similarly, the best practices doc suggests scaffolding your tests by running your deploy script here, that way you're testing against the post-deploy state. Of course, not everyone does this, and it's not always applicable. But this is inconsistent with what's recommended, so wonder what the best approach is here. Perhaps we should add a comment linking to the best practices or something?

Reply:
I have no comments on this. I will go with whatever the team here decides. As an aside, If we take any action, we may need to update the default `CounterTest` file too:

https://github.com/foundry-rs/foundry/blob/42beef0a1d6fc4a26a803e6321e041215123753e/cli/assets/CounterTemplate.t.sol#L8-L11

Reply:
I don't have a good generic solution here, so if we're in favor of adding this command we can leave this as-is. Will resolve this :)